### PR TITLE
Release Notes for 2.10

### PR DESCRIPTION
**Pull request by JustinSGray**

### CODE REVIEW

**Comment by hschilling** [comment]
In the line about

 <POEM 012>The old implementation was hard coded to the `gesdd` method for inverses. 
  Its now an option, but we've changed to `gesvd`, which might introduce slight 
  numerical changes to the kriging fits.

Can you say the old implementation of KrigingSurrogate ?

**Comment by hschilling** [comment]
The line

The N2 viewer now works, even for models that are run in parallel (i.e. if your using mpi to call python)

makes it sound like N2 didn't work before at all ! Maybe just say

The N2 viewer now works for models that are run in parallel (i.e. if your using mpi to call python)

File: release_notes.txt [hunk 1]
```diff
@@ -1,3 +1,97 @@
...
+  global_size of distributed outputs in their reports
+
+
+Bug Fixes: 
+----------
+- Fixed a hang in SimpleGA driver related to mpi calls 
+- Fixed a bug in SimpleGA related to setting a vector of constraints
+- Various bug-fixes to allow running with Python 3.8
+- Fixed a bug in ScipyOptimizer/Cobyla related to objective and dv scaling
+- Corrected bug prenting the user from passing a list to `set_val` when the variable is an array
```
Comment (Kenneth-T-Moore) [inline release_notes.txt]: prenting -> preventing

File: release_notes.txt [hunk 2]
```diff
@@ -1,3 +1,97 @@
...
+
+Backwards Incompatible API Changes: 
+-----------------------------------
+- <POEM 001> updated definition of light year to correct IAU definition 
+  (very minor change to unit conversions associated with this update)
+
+- <POEM 004> AkimaSplineComp and BsplineComp have been deprecated, replaced with SplineComp
+
+- <POEM 008>
+  - The user is not requires to set a value for the solver option `solve_subsystems`
```
Comment (Kenneth-T-Moore) [inline release_notes.txt]: requires -> required
not -> now

File: release_notes.txt [hunk 3]
```diff
@@ -1,3 +1,97 @@
...
+New Features: 
+-------------
+
+- <POEM 001> astronmical unit (au) and parsec (pc) added to the standard unit definitions. 
+
+- <POEM 002> users can now manually terminate an optimization using operating system signals 
+
+- <POEM 003> 
+  - users can now issue I/O, connections, and promotions in configure 
+  - there was a new API method added to groups to allow promotions after a subsystem has already been added
```
Comment (Kenneth-T-Moore) [inline release_notes.txt]: there was a new API method added -> new API method added

File: release_notes.txt [hunk 4]
```diff
@@ -1,3 +1,97 @@
...
+
+- <POEM 004> SplineComp was added to the standard component library, 
+  and you can also import the spline utilities to use in your own components
```
Comment (Kenneth-T-Moore) [inline release_notes.txt]: spline utilities -> interpolation algorithms

File: release_notes.txt [hunk 5]
```diff
@@ -1,3 +1,97 @@
...
+
+- Improvement to the bidirectional Jacobian coloring so it will fall back to 
+  pure forward or reverse if thats faster
+
+- Users can now set the shape for outputs of `BalanceComp` and `EqConstraintsComp`
+
+- Improvements to `debug_print` for driver related to reporting which derivatives 
+  are being computed when coloring is active
+
+- The N2 viewer now works, even for models that are run in parallel (i.e. if your using mpi to call python)
```
Comment (Kenneth-T-Moore) [inline release_notes.txt]: your -> you're

**Comment by swryan** [comment]
https://github.com/JustinSGray/openmdao/pull/11

**Pull request merged**